backend/routes/profile.py

```
@@ -0,0 +1,109 @@
+import os
+import uuid
+from pathlib import Path
+import secrets
+
+router = APIRouter()
+
+# ✅ Predefined fields for validation
+RELEVANT_FIELDS = [
+    "Computer Science", "Electrical Engineering", "Mechanical Engineering",
+    "Civil Engineering", "Artificial Intelligence", "Data Science", "Robotics",
+    "Physics", "Mathematics", "Economics", "Biotechnology"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)